utils/utils_test.go

```diff
@@ -0,0 +1,89 @@
+package utils
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseCommandLine(t *testing.T) {
+	t.Parallel()
+
+	type testCase struct {
+		name            string
+		input           string
+		expectedSuccess []string
+		expectedError   string
+	}
+
+	testCases := []testCase{
+		{
+			name:            "empty input",
+			input:           "",
+			expectedSuccess: []string{},
+			expectedError:   "",
+		},
+		{
+			name:            "single argument",
+			input:           "arg1",
```

Comment:
isn't the first word the command? e.g. `command arg1 arg2`?

Reply:
Also, can the tests be more aligned with reality? do we normally do `commandname --arg1=value1`?